rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc

```diff
@@ -895,7 +895,7 @@ include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
 [%header,cols="2a,1a"]
 |===
 |Issue Resolution |ID
-| RPA Manager no longer includes the _userName_ and _password_ fields for the _AWS Credentials_ type in the OpenAPI Specification (OAS) when you publish an invokable run configuration to Anypoint Exchange. | W-12141693
+| RPA Manager no longer includes the _userName_ and _password_ fields for the _AWS Credentials_ type in the OpenAPI Specification (OAS) when you publish an invocable run configuration to Anypoint Exchange. | W-12141693
```

Comment:
Our style is to bold for UI elements (assuming the fields are in the UI.) If the RPA docs have been using backticks, you should keep it consistent.

Reply:
References the OAS, not a UI element -> not changed.